### PR TITLE
Fixed Default Edit

### LINKED ISSUE #398: Input Default Editable
image
- The input component for the forms and user dashboard are the same
- You should give Input another prop to indicate if it is editable by default
- make sure both register form and user dashboard works

### PR DESCRIPTION
**Pull request by rfairooz**
Dashboard works like before
Screenshot 2023-08-25 at 1 58 35 PM
Screenshot 2023-08-25 at 1 58 15 PM
All Forms are now editable by default.

Screenshot 2023-08-25 at 2 01 52 PM

Addressed Comments

### CODE REVIEW

**Comment by shahdivyank** [comment]
can you make the default in the true in the params and then if its false then specify. It seems to be true in most cases so in the component header of the input u can do
```
const Input = ({ something, something, editable = true }) => {
``

File: src/components/Forms/Register.jsx [hunk 1]
```diff
@@ -55,6 +55,7 @@ const Register = () => {
               maxLength={50}
               user={user}
               setUser={setUser}
+              editable={true}
```
Comment (shahdivyank) [inline src/components/Forms/Register.jsx]: you can remove these lines now that the default is true. make sure to say its false in case u need that

Comment (rfairooz) [reply src/components/Forms/Register.jsx]: ah shoot sorry missed that one

Comment (rfairooz) [reply src/components/Forms/Register.jsx]: ok fixed register. All other forms alr fixed in prev commit

**Review by shahdivyank — APPROVED** [review]

**Pull request merged**